### DISCUSSION TITLE
커버 아이텀 버그 및 스킨 사용중에 사이드바 질문 드립니다.

### DISCUSSION THREAD

**Discussion by chohs228** [pronist/hELLO#81] [Q&A] [answered]
안녕하세요. 기본 스킨 사용 중에 hELLO 스킨으로 변경 하여 사용 하던 중 사용 하던 기능이 없어져서 문의드립니다.
관리 페이지에 보면 링크가 있고, 거기 추가한 링크를 사이드바에서 이용 가능했는데 해당 스킨은 사이드바에 링크가 없더라구요.
혹시 이 부분을 차후 기능에 추가 해주실 계획이 있으신가요?
혹은 추가 계획이 없다면 파일의 어느 부분에 추가 해야 하는지 알려주실 수 있을까요?

또한 커버 아이템 - 콘텐츠 직접입력 시 홈페이지에 내용 출력이 안됩니다.

**Comment by pronist** [comment, marked as answer]
@chohs228 

1. 링크 기능 관련해서는 현재 [티스토리 스킨개발가이드](https://tistory.github.io/document-tistory-skin/) 에 나와있지 않습니다. 예전에 있던 기능인데, 이 부분이 스킨가이드에 나와있지 않다는 것은 암묵적으로 링크기능이 사라질 것이고 문제가 발생할 수도 있어 포함시키지 않습니다. 캘린더 기능도 마찬가지죠.

이 기능을 추가시키려면 기존의 스킨, ```skin.html``` 에서 ```s_sidebar``` 에 있는 링크기능을 찾고 ```s_sidebar_element``` 항목으로 추가시켜주어야 합니다.

2.  버그에 대한 내용은 이슈를 등록할 떄 나오는 버그 등록 템플릿에 따라 등록 부탁드립니다.
